bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml

```diff
@@ -325,14 +325,6 @@ spec:
           - get
           - patch
           - update
-        - apiGroups:
-          - security.openshift.io
-          resourceNames:
-          - anyuid
-          resources:
```

Comment:
why removed?

Reply:
The permission was moved to a Role below instead of a ClusterRole to reduce permissions.